server/src/services/job.service.ts

```diff
@@ -39,8 +39,7 @@ const asJobItem = (dto: JobCreateDto): JobItem => {
 @Injectable()
 export class JobService extends BaseService {
   @OnEvent({ name: 'config.init' })
-  @OnEvent({ name: 'config.update', server: true })
-  onConfigInitOrUpdate({ newConfig: config }: ArgOf<'config.init'>) {
+  onConfigInit({ newConfig: config }: ArgOf<'config.init'>) {
     if (this.worker !== ImmichWorker.MICROSERVICES) {
       return;
     }
```

Comment:
You can get rid of this condition now because it'll only get called on microservice instances anyways (since there isn't `server: true`)

Reply:
No, this will run on both API and Microservices when config init runs